modules/search/class.jetpack-search-widget-filters.php

```diff
@@ -11,6 +11,18 @@ function __construct() {
 			return;
 		}
 
+		/**
+		 * Enables or disables the search filters widget. Disabled by default for Jetpack sites.
+		 * Enable if your site's security strategy allows aggregation.
+		 *
+		 * @since 5.5
```

Comment:
Could you change that to 5.5.0?

Could you also add `* @module search` beforehand? Here is a good example:
https://github.com/Automattic/jetpack/blob/5.4/modules/search/class.jetpack-search.php/#L282

Thanks!